synapse/rest/client/relations.py

```diff
@@ -60,17 +59,7 @@ async def on_GET(
         #
         # TODO Use PaginationConfig.from_request when the unstable parameter is
         #      no longer needed.
-        direction = parse_string(request, "dir", allowed_values=["f", "b"])
-        if direction is None:
-            if self._msc3715_enabled:
-                direction = parse_string(
-                    request,
-                    "org.matrix.msc3715.dir",
```

Comment:
To check: this is an unstable query parameter that's been stabilised, and we're now requiring the use of the stable version?

Reply:
Correct.